qml/items/FreeCellItem.qml

```diff
@@ -0,0 +1,41 @@
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+
+Rectangle {
+    id: "item"
+    property int stack;
+    property int maxStack;
+
+    function canReleaseCard()
+    {
+        return true;
+    }
+
+    function acceptsSuit(suit)
+    {
+        return item.acceptedSuits.indexOf(suit) !== -1
+    }
+
+    function acceptsRank(rank) {
+        return true;
+    }
+
+    function freeFieldCells()
+    {
+        return Qt.freeFieldCells; // overwritten in FieldCell
+    }
+
+    function maxMoveStack(target)
```

Comment:
This is where the new logic for maximum movable stack size is.
Firstly, we delegate to the target to determine the number of free field cells (aka columns) -- if the target itself is a fieldCell, the global value is reduced by one as it cannot store a temporary stack itself.
This would be the place to honor the player preference mentioned in your comment on issue #6 
For "pessimistic mode" (the status quo) you would return Qt.freeSingleCells + target.freeFieldCells()